addons/sourcemod/scripting/CallAdmin.sp

```diff
@@ -335,18 +340,68 @@ stock int GetTeamAliveCount(int team)
 
 public void OnWebHookExecuted(HTTPResponse response, DataPack pack)
 {
-	int client = pack.ReadCell();
-	delete pack
+	static int retries;
+	
+	pack.Reset();
+	int userid = pack.ReadCell();
+	int client = GetClientOfUserId(userid);
+	Webhook hook = view_as<Webhook>(pack.ReadCell());
 
 	if (!client || !IsClientInGame(client))
+	{
+		delete hook;
+		delete pack;
 		return;
+	}
 
 	if (response.Status != HTTPStatus_OK)
 	{
-		CPrintToChat(client, "%s {red}Failed to send your message.", CHAT_PREFIX);
+		if(retries < 3)
+			CPrintToChat(client, "%s {red}Failed to send your message. Resending it .. (%d/3)", CHAT_PREFIX, retries);
+		
+		else if(retries >= 3)
+		{
+			CPrintToChat(client, "%s {red}An error has occurred. Your message can't be sent.", CHAT_PREFIX);
+			LogError("[CallAdmin] Message can't be sent after %d retries.", retries);
+			delete hook;
+			delete pack;
+			return;
+		}
+		
+		char webhookURL[PLATFORM_MAX_PATH];
+		pack.ReadString(webhookURL, sizeof(webhookURL));
+		
+		DataPack newPack;
+		newPack.WriteCell(userid);
+		newPack.WriteCell(view_as<int>(hook));
+		newPack.WriteString(webhookURL);
+		CreateDataTimer(0.5, ExecuteWebhook_Timer, newPack);
```

Comment:
mb you can put back this function before calling writecell

i didnt know that CreateDataTimer was actually creating the datapack:
- https://github.com/alliedmodders/sourcemod/blob/6e73aba2500f1565464955f48f17a4ed52da2a5c/plugins/include/timers.inc#L195